frontend/src/App.vue

```diff
@@ -1,16 +1,18 @@
 <script setup lang="ts">
+import IncomeStatement from "./components/IncomeStatement.vue";
+import Auditor from "./components/Auditor.vue";
+import Expenses from "./components/Expenses.vue";
 </script>
 
 <template>
-  <header>
-  </header>
-
-  <main>
-    <h1 class="text-5xl font-bold underline">
-      Hello world!
-    </h1>
+  <main class="m-10 border border-sky-500">
+    <div class="mt-8 ml-3 mb-3 text-xl">
+        INCOME STATEMENT
+    </div>
+    <div class="grid grid-cols-2">
+      <IncomeStatement/>
+      <Auditor class="row-span-2"/>
```

Comment:
Never put classes directly on components, think about how to structure this better.